scripts/postbuild.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

this is more "proper", and your version also wouldn't work in nixos